### ISSUE TITLE
Name of package in conflict with directory module when building for remote

### ISSUE THREAD

**Issue by samedii** [guildai/guildai#159]
If the package has the same name as a directory with a `__init__.py` then you get this error when building the package for running remote.

```
- package: ampli-model

- model: ampli
  sourcecode:
    - '*.py'
    - guild.yml
...
```

```
(venv) samedii@tricky:~/Documents/ampli-model$ guild run retrain model=dff339b4 --stage --label bce-every3-emphasize2 -r vastai
WARNING: cannot find a suitable run for required resource 'model'
You are about to stage ampli-model:retrain on vastai
  batch_size: 100
  ...
Continue? (Y/n) 
Building package
package src: /home/samedii/Documents/ampli-model
package dist: /tmp/guild-remote-stage-v8tq8bwq
running clean
removing 'build/lib' (and everything under it)
removing 'build/bdist.linux-x86_64' (and everything under it)
'build/scripts-3.8' does not exist -- can't clean it
removing 'build'
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/ampli_model
copying /home/samedii/Documents/ampli-model/ampli_model/loss.py -> build/lib/ampli_model
copying /home/samedii/Documents/ampli-model/ampli_model/__init__.py -> build/lib/ampli_model
copying /home/samedii/Documents/ampli-model/ampli_model/settings.py -> build/lib/ampli_model
error: package directory '/home/samedii/Documents/ampli-model/ampli_model/ampli_model' does not exist
```

**Comment by gar1t** [comment]
This issue is fixed in master by detecting the condition and existing with a meaningful error message.

This conflict is legit as the Guild package has to be installed alongside the other locally defined Python packages - the names cannot conflict.

Thanks for taking the time to work through this!

**Comment by gar1t** [comment]
This should be fixed in 0.7.0. If you still see the behavior, please re-open. Thanks again for the report!
